sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/Generated/AadPropertiesOperationsExtensions.cs

```diff
@@ -44,12 +48,15 @@ public static AADPropertiesResource Get(this IAadPropertiesOperations operations
             /// <param name='azureRegion'>
             /// Azure region to hit Api
             /// </param>
+            /// <param name='odataQuery'>
+            /// OData parameters to apply to the operation.
+            /// </param>
             /// <param name='cancellationToken'>
             /// The cancellation token.
             /// </param>
-            public static async Task<AADPropertiesResource> GetAsync(this IAadPropertiesOperations operations, string azureRegion, CancellationToken cancellationToken = default(CancellationToken))
+            public static async Task<AADPropertiesResource> GetAsync(this IAadPropertiesOperations operations, string azureRegion, CancellationToken cancellationToken = default(CancellationToken), ODataQuery<BMSAADPropertiesQueryObject> odataQuery = default(ODataQuery<BMSAADPropertiesQueryObject>))
```

Comment:
This looks like it was edited after generation, was it?  Generally, to work around braking changes like this, you write a customization to implement the old method signature.  You can find an example of this here: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations/VirtualMachineScaleSetVMsOperationsExtensions.cs

Note that you will need to make this change in the interface, the implementing class, and the extension for this method